content/docs/06-integrations/00-longhorn.md

```diff
@@ -0,0 +1,41 @@
+---
+title: 'Longhorn'
+metaTitle: 'Longhorn Integration with Palette'
+metaDescription: 'Longhorn pack in Palette'
+hiddenFromNav: true
+isIntegration: true
+category: ["storage"]
+logoUrl: 'https://registry.spectrocloud.com/v1/csi-longhorn/blobs/sha256:8257bd6697941139cea8ace907e25b3859cb8de48f965a5b6011d518cad0a2db?type=image/png'
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+# Longhorn Overview
+
+Longhorn is a lightweight distributed block storage system for cloud native storage Kubernetes that allows you to replicate storage to Kubernetes clusters. You can install Longhorn on an existing Kubernetes cluster with one ``kubectl apply`` command or using Helm charts. Once Longhorn is installed, it adds persistent volume support to the Kubernetes cluster using containers and microservices.
+
+Longhorn creates a dedicated storage controller for each block device volume and replicates the volume across multiple nodes. 
+
+# Prerequisites
+
+- Kubernetes cluster is at least version 1.18 and not higher than 1.24.
```

Comment:
Min version should be 1.22 as Palette stopped supporting older versions of k8s

Reply:
@brathina-spectro Fixed